src/main/java/gregtech/api/metatileentity/MetaTileEntityHolder.java

```diff
@@ -277,6 +277,7 @@ public void receiveInitialSyncData(PacketBuffer buf) {
         if (buf.readBoolean()) {
             int metaTileEntityId = buf.readVarInt();
             setMetaTileEntity(GregTechAPI.MTE_REGISTRY.getObjectById(metaTileEntityId));
+            this.metaTileEntity.onPlacement();
             this.metaTileEntity.receiveInitialSyncData(buf);
             scheduleRenderUpdate();
             this.needToUpdateLightning = true;
```

Comment:
This whole block and one below in this file are near identical (except param names) so you could extract it out.